engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/SystemProcessTest.scala

```diff
@@ -55,7 +55,8 @@ class SystemProcessTest extends InterpreterTest with OsSpec {
           |""".stripMargin
 
       val error = the[InterpreterException] thrownBy eval(code)
-      error.getMessage should include("java.io.IOException")
```

Comment:
@hubertp this one is for you to review. Originally the `PanicException` used `InteropLibrary` to invoke `to_display_text` - obviously `IOException` wrapped in host object didn't have it, so the system fallen back to `typeToDisplayTextNode.execute(payload)`.

I am changing the code to use `IndirectInvokeMethodNode` to invoke the method. That means it is going to fallback to `Any.to_display_text` and that one properly extracts the message from the `IOException`.

I am modifying the test to check a part of the message is there, but the type of the exception isn't.

Reply:
That sounds OK'ish.